### DISCUSSION TITLE
popover but content stays in dom?

### DISCUSSION THREAD

**Discussion by dadamssg** [radix-ui/primitives#2022] [Help] [answered]
I'm trying to implement a combobox of sorts and was using the `<Popover />` component but then realized that my form elements were getting wiped from the dom when it's in the closed state. 

<img width="352" alt="image" src="https://user-images.githubusercontent.com/881986/225057377-c28cd6d4-01d5-4145-8353-f6d064d19231.png">

I need these checkboxes to exist in the dom even after the popover is closed. I noticed the `forceMount` prop on `Popover.Portal` but using that and hiding it based on `data-state="closed"` breaks the tab navigation and makes it wonky. 

Is there a way to achieve what i want with radix?

**Comment by benoitgrelard** [comment, marked as answer]
It's not currently possible, here are some related discussions if you want to dig deeper:
- #1155
- #855
